### ISSUE TITLE
FR: Scroll To or Search For Installed Plugin & Search In Community Plugins & Theme browsers

### ISSUE THREAD

**Issue by Graywaren** [Vinzent03/obsidian-advanced-uri#193]
Hi
I  love that I can open a plug-ins setting menu or the community plug-ins tab as well as open the browser for community plugins and themes. I'm currently creating a script that generates a list of all installed plugins and I really want to link them to the settings menu, however this doesn't work that well with the current link methods since if a plug-in does not have its own settings page, the link just opens the settings menu to whatever it was open to last. My only choice would be to set them all to just open the community plugins tab and need to search for it.

I'd really love to see the ability to:
1. have the link to the community plugin, plug-in browser, and theme browser pages, input text into the search box.  Something like: 
`obsidian://advanced-uri?vault=<vault>&settingid=<plugin id>search=<plugin name>` 
2. In the case of the community plugin tab, have it be able to scroll down to that plugin in the installed list like:
`obsidian://advanced-uri?vault=<vault>&settingid=community-plugins&settingsection=<plugin name>`
2. This seems possibly too complicated to implement, but have links to community plug ins that don't have their own settings page default to the above search or scrolling behavior or even just to the community plugins tab so:
`obsidian://advanced-uri?vault=<vault>&settingid=<plugin id>` would redirect to `obsidian://advanced-uri?vault=<vault>&settingid=<plugin id>search=<plugin name>` if there was not a setting id page for that plug-in id.

Just the ability to have it already search the plugin/theme name in those areas would be fantastic and solve my conundrum of dynamically generated links

Thanks for making such a great plugin!

**Comment by Thundercraft5** [comment]
I agree, this would be very helpful for creating a template vault which would allow for a quick way to selectively install some plugins on from a list of recommended plugins.
